### PR TITLE
Recommendation to use v2 Feeds

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] Ready

## Description
Recommendation to use v2 MITRE & Unit42 Feeds.

## Related Issue 
related: https://github.com/demisto/etc/issues/40624#issuecomment-908134312

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/FeedUnit42/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+We recommend using the Unit42 v2 Feed instead. The Unit42 ATOMs Feed is more aligned with the STIX format, and includes more objects, such as Campgain, Course of Action and Relationships.
```
Comment (ytauman) [inline Packs/FeedUnit42/README.md:1]: didn't we say we wanted to change it to 'Unit42 ATOMs'?

Comment (ytauman) [reply Packs/FeedUnit42/README.md:1]: plus typo on campaign (my bad haha)

File: Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ configuration:
   required: false
   type: 8
 description: Unit42 feed of published IOCs, which contains known malicious indicators.
-display: Unit42 v2 Feed
+display: Unit42 ATOMs Feed
```
Comment (ytauman) [inline Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.yml:97]: so we decided to change the name in this PR? sweet 👌

**Review by ytauman — APPROVED** [review]

**Review by yuvalbenshalom — APPROVED** [review]

**Comment by ChanochShayner** [comment]
added the doc approved tag as this was being written by @ytauman 
@ytauman please confirm.

**Comment by ytauman** [comment]
> added the doc approved tag as this was being written by @ytauman
> @ytauman please confirm.

already approved :)

**Pull request merged**